spec/util/test.js

```diff
@@ -76,5 +76,9 @@ module.exports.beforeEach = function(testCase, env) {
         });
         env.mockAsapiController = env.asapiMock.create();
     }
+
+    process.on("unhandledRejection", function(reason, promise) {
+        throw new Error("Unhandled rejection: " + reason);
```

Comment:
I worry this will stop any useful stack traces. Can you confirm stack traces are still okay?